packages/client/src/components/pages/muster-page/muster-page.tsx

```diff
@@ -599,69 +598,28 @@ export const MusterPage = () => {
             </Grid>
           )}
 
-          {/* Weekly Non-Compliance */}
-          <Grid item xs={6}>
-            <Card>
-              <CardContent>
-                <Typography variant="h5" gutterBottom>
-                  Weekly Non-Compliance
-                </Typography>
-
-                <div>
-                  <span>Viewing the top </span>
-                  <Select
-                    className={classes.trendCountSelect}
-                    value={weeklyTrendTopUnitCount}
-                    onChange={e => setWeeklyTrendTopUnitCount(parseInt(e.target.value as string))}
-                  >
-                    {Array.from(Array(maxTopUnitsCount).keys()).map(value => (
-                      <MenuItem key={value} value={value + 1}>{value + 1}</MenuItem>
-                    ))}
-                  </Select>
-                  <span> offending units.</span>
-                </div>
-
-                <Plot
-                  style={{ width: '100%' }}
-                  data={weeklyTrendData}
-                  layout={weeklyTrendChart.layout}
-                  config={weeklyTrendChart.config}
-                />
-              </CardContent>
-            </Card>
-          </Grid>
-
-          {/* Monthly Non-Compliance */}
-          <Grid item xs={6}>
-            <Card>
-              <CardContent>
-                <Typography variant="h5" gutterBottom>
-                  Monthly Non-Compliance
-                </Typography>
-
-                <div>
-                  <span>Viewing the top </span>
-                  <Select
-                    className={classes.trendCountSelect}
-                    value={monthlyTrendTopUnitCount}
-                    onChange={e => setMonthlyTrendTopUnitCount(parseInt(e.target.value as string))}
-                  >
-                    {Array.from(Array(maxTopUnitsCount).keys()).map(value => (
-                      <MenuItem key={value} value={value + 1}>{value + 1}</MenuItem>
-                    ))}
-                  </Select>
-                  <span> offending units.</span>
-                </div>
-
-                <Plot
-                  style={{ width: '100%' }}
-                  data={monthlyTrendData}
-                  layout={monthlyTrendChart.layout}
-                  config={monthlyTrendChart.config}
-                />
-              </CardContent>
-            </Card>
-          </Grid>
+          {/* Muster compliance Table */}
```

Comment:
It looks like this is a React component which could be extracted out into a separate file.

Reply:
See comment above